service/history/timerQueueTaskProcessor.go

```diff
@@ -0,0 +1,322 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
plz also port the existing tests to `timerQueueTaskProcessor_test.go`

Reply:
done